spec/jobs/operation_job_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe OperationJob do
+  let(:operation) do
+    create(:operation,
+      job_class: subject,
+      job_id: nil,
+      job_started_at: nil,
+      job_completed_at: nil)
```

Comment:
Please change the factory to have these things as nil as defaults. It doesn't make sense to me to have values for them since they are usually nil at the beginning.